mne/viz/evoked.py

```diff
@@ -1191,3 +1191,350 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     # show and return it
     plt_show(show)
     return fig
+
+
+def _ci(arr, ci):
+    """Calculate the `ci`% parametric confidence interval for `arr`.
+    Aux function for plot_compare_evokeds."""
+    from scipy import stats
+    mean, sigma = arr.mean(0), stats.sem(arr, 0)
+    # This is highly convoluted to support 17th century Scipy
+    # XXX Fix when Python 2.6 support is dropped!
+    return np.asarray([stats.t.interval(ci, arr.shape[0],
+                       loc=mean_, scale=sigma_)
+                       for mean_, sigma_ in zip(mean, sigma)]).T
+
+
+def _setup_styles(conditions, style_dict, style, default):
+    """Aux function for plot_compare_evokeds to set linestyles and colors"""
+    condition_warning = "Condition {0} could not be mapped to a " + style
+    style_warning = ". Using the default of {0}".format(default)
+    for condition in conditions:
+        if condition not in style_dict:
+            if "/" not in condition:
+                warn(condition_warning.format(condition) + style_warning)
+                style_dict[condition] = default
+            for style_ in style_dict:
+                if style_ in condition.split("/"):
+                    style_dict[condition] = style_dict[style_]
+                    break
+    return style_dict
+
+
+def plot_compare_evokeds(evokeds, picks=None, conditions=None, ch_names=None,
+                         colors=None, linestyles=['-'], styles=None,
+                         vlines=[0], ci=0.95, truncate_yaxis=True, ymin=None,
+                         ymax=None, invert_y=False, ax=None, title=None,
+                         show=True):
+    """ Plot, for multiple conditions, individual evoked time courses of one
+    sensor or of the mean of multiple sensors
+
+    This function is useful for comparing ER[P/F]s at a specific location. It
+    plots Evoked data or, if supplied with a list/dict of lists of evoked
+    instances, grand averages plus confidence intervals.
+
+    Parameters
+    ----------
+    evokeds : instance of mne.Evoked | list | dict
+        If a single evoked instance, it is plotted as a time series.
+        If a dict whose values are Evoked objects, the contents are plotted as
+        single time series each and the keys are used as condition labels.
+        If a list of Evokeds, the contents are plotted with indices as labels.
+        If a [dict/list] of lists, the unweighted mean is plotted as a time
+        series and the parametric confidence interval is plotted as a shaded
+        area. All instances must have the same shape - channel numbers, time
+        points etc.
+    picks : int | list of int | str | list of str | None
+        The indices (int) or names (str) of the sensors to average and plot.
+        Must all be of the same channel type.
+        If the selected channels are gradiometers, the corresponding pairs
+        will be selected.
+        If None, the global field power is plotted instead.
```

Comment:
Not necessarily two lines:
```
import mne
...
mne.viz.plot_combine_evoked(a_dict, picks='Cz')
```
vs
```
import mne
...
mne.viz.plot_combine_evoked(a_dict, picks=pick_channels(ev.ch_names, ['Cz']))
```
I agree yours is simpler, but I find it hard to justify why this function should be the exception to the rule that is uniform everywhere else. In my mind it makes it *harder* for people to meaningfully use the package instead of *easier*. But if everyone else still thinks this is a good plan despite this consistency argument, I can live with it.